### ISSUE TITLE
W[4]-Submission

### ISSUE THREAD

**Issue by hmaskai** [hmaskai/TwitterNew#4]
I have included the count change part, please review my application. /cc @codepathreview

**Comment by avo1** [comment]
:+1: Nice work! The point of this homework was to explore an example of a full MVC application with a RESTful API. 

Good job implementing the fav/retweet number update with nice button images. However, you haven't implemented the API call for that function. You need to call the API for [Retweeting](https://dev.twitter.com/rest/reference/post/statuses/retweet/%3Aid)  and [favoriting](https://dev.twitter.com/rest/reference/post/favorites/create) to actually perform those actions. If you mimic the Twitter app, you should be able to fav only once. Tap on fav 1 more time, it will become un-fav (and decrease the number by 1, but this is not required - it's in the optional tasks)

We have a detailed [Project 4 Feedback Guide](http://courses.codepath.com/snippets/ios_university/feedback_guides/project_4_feedback.md) which covers the best practices for implementing this assignment. Read through the feedback guide point-by-point to determine ways you might be able to improve your submission. You should consider going back and implementing these improvements as well. Keep in mind that one of the most important parts of iOS development is learning the correct patterns and conventions.

If you have any particular questions about the assignment or the feedback, feel free to reply here or email us at universitysupport@codepath.com.

Next time please reply on original issue to get my immediate attention, do not create new issue.

Regards,
Dave
